gensim/models/fasttext_inner.pyx

```diff
@@ -243,6 +243,37 @@ cdef void fast_sentence_cbow_hs(
             our_saxpy(&size, &word_locks_ngrams[subwords_idx[m][d]], work, &ONE, &syn0_ngrams[subwords_idx[m][d]*size], &ONE)
 
 
+cdef init_ft_config(FastTextConfig *c, model, alpha, _work, _neu1):
+    c[0].hs = model.hs
```

Comment:
why `c[0]` instead of `*c` ?

Reply:
https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#types

> Note that Cython uses array access for pointer dereferencing, as *x is not valid Python syntax, whereas x[0] is.